v1/api/index.ts

```diff
@@ -43,7 +46,7 @@ const v1Routes = (cfg: Partial<APIConfig>, store: StoreI) => async (server: Fast
 
   if (cfg.useSwagger ?? false) {
     await server.register(swagger, {
-      swagger: {
+      openapi: {
```

Comment:
What's the difference?

Reply:
Does this mean we need to use a different api generator thing? cc @fauno

Reply:
I'll try both!